index.ts

```diff
@@ -60,6 +60,14 @@ module ReactNativeBiometrics {
      * Enum for generic biometrics (this is the only value available on android)
      */
     export const Biometrics = 'Biometrics';
+    /**
+     * Enum for android's predefined error cases  
+     */
+    export const BiometricErrorAndroid = {
```

Comment:
@whitedogg13 

1) where do you return these enum values in the code? I can't see any change about that?
2) is it not better to change the promise `error` type from string to this enum for having better type checking?
3) you said you tried to define an enum according to the above doc, but you already defined a const

```tsx
export enum BiometricErrorAndroid {
  NO_HARDWARE: "BIOMETRIC_ERROR_NO_HARDWARE",
  HW_UNAVAILBLE: "BIOMETRIC_ERROR_HW_UNAVAILABLE",
  NONE_ENROLLED: "BIOMETRIC_ERROR_NONE_ENROLLED",
}
```

Reply:
I think this feature has been requested by a few different people. I am going to slate this for either #183 or a future release.

Reply:
I think the intent here was to expose an enum from the library for being able to easily match on the strings we return for the sensor error states. You can see them here: https://github.com/SelfLender/react-native-biometrics/blob/master/android/src/main/java/com/rnbiometrics/ReactNativeBiometrics.java#L68

I still think we need to go further to create an ENUM on the Java side to ensure these strings are not messed with. Really it would be ideal to have the Java ENUM inform the Typescript ENUM. I am just not sure how to go about that at this moment.